operator/controllers/mlserver.go

```diff
@@ -197,6 +194,10 @@ func getMLServerEnvVars(pu *machinelearningv1.PredictiveUnit) ([]v1.EnvVar, erro
 			Name:  MLServerModelURIEnv,
 			Value: DefaultModelLocalMountPath,
 		},
+		{
+			Name:  MLServerTempoRuntimeEnv,
```

Comment:
Would it make sense to allow user to also provide overrides? or is that already covered by the podspec itself?

Reply:
yes, replicas could be done by sdep itself.